pkg/vdri/trustbloc/vdri.go

```diff
@@ -61,11 +61,14 @@ type VDRI struct {
 	authToken        string
 
 	validatedConsortium map[string]bool
+
+	enableSignatureVerification bool
```

Comment:
as a followup, we should think about how to handle the other verification option (fetching matching document from sufficient members). we also need to discuss the default.

verification methods:
- none
- signature
- match